src/components/NewWorkspaceModal.js

```diff
@@ -210,6 +222,17 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
           options: _.sortBy('label', allRegions)
         })
       ])]),
+      shouldShowDifferentRegionWarning() && div({ style: { ...warningStyle } }, [
+        icon('warning-standard', { size: 24, style: { color: colors.warning(), flex: 'none', marginRight: '0.5rem' } }),
+        div({ style: { flex: 1 } }, [
+          `Copying data from `,
+          span({ style: { fontWeight: 600 } }, [`${getRegionInfo(sourceWorkspaceLocation, sourceLocationType).regionDescription} `]),
+          `to `,
+          span({ style: { fontWeight: 600 } }, [`${getRegionInfo(bucketLocation, destLocationType).regionDescription} `]),
+          `may incur network egress charges. `,
```

Comment:
For A11y, it is best to use a `strong` tag rather than manually embolden the text.
Also, there are a few strings that do not need interpolation.

```suggestion
          strong([getRegionInfo(sourceWorkspaceLocation, sourceLocationType).regionDescription]),
          ' to ',
          strong([getRegionInfo(bucketLocation, destLocationType).regionDescription]),
          ' may incur network egress charges. ',
```

Reply:
Done.